api/goldens/aptos_api__tests__transactions_test__test_get_txn_execute_failed_by_entry_function_execution_failure.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
I wonder if we need this golden file at all. It becomes painful having to keep this up to date.